src/uploader.py

```diff
@@ -42,21 +40,24 @@ def run(self, trigger_file=None):
         self.logger.info(logfiles)
         for f in logfiles:
             self.logger.info("parsing {}".format(f))
+            in_file = self.s3.get_logfile(f)
+            logfile_content = in_file.read()
+            self.logger.iinfo("read {} bytes".format(sys.getsizeof(logfile_content)))
             for parser in self.parsers:
                 try:
-                    self.etl(parser, f)
+                    self.logger.info("Using {}".format(parser.__class__.__name__))
```

Comment:
Not a big deal, but I tend to put log messages like that inside the method so that you always get consistent logs no matter who the caller is. Here that would be inside `etl()`. Doesn't make much difference in this case since there's only one caller.